test/test_chatcommands.py

```diff
@@ -172,7 +172,7 @@ def test_report(handle_spam):
 
 
 @patch("chatcommands.handle_spam")
-def test_checkpost(handle_spam):
+def test_scan(handle_spam):
```

Comment:
Maybe add test cases for scanning multiple posts?

Reply:
Nah, not necessary at this point. I'm going to merge `scan` with `report` as said in the last paragraph of the PR description. I'll add tests then.